package.json

```diff
@@ -44,7 +44,7 @@
     "mini-css-extract-plugin": "0.8.0",
     "minimatch": "3.0.4",
     "node-fetch": "2.6.0",
-    "node-sass": "4.12.0",
+    "node-sass": "^4.13.0",
```

Comment:
I think that the convention is to use the fixed version for the dependencies

Reply:
We use package-lock anyway, so it doesn't matter much on practice (and `^` would make updates a bit easier), but given that all other packages are specified without `^`, I can change back to match it.

Reply:
As you want, I only saw something that caught my attention ... it is a small detail without importance.

Btw, nice job 😊